app/assets/javascripts/components/features/ui/components/onboarding_modal.jsx

```diff
@@ -25,8 +25,8 @@ const PageOne = ({ acct, domain }) => (
 
     <div>
       <h1><FormattedMessage id='onboarding.page_one.welcome' defaultMessage='Welcome to Mastodon!' /></h1>
-      <p><FormattedMessage id='onboarding.page_one.federation' defaultMessage='Mastodon is a social network that belongs to everyone.' /></p>
-      <p><FormattedMessage id='onboarding.page_one.handle' defaultMessage='You are on {domain}, one of many independent Mastodon instances. Your full handle is {handle}' values={{ domain, handle: <strong>{acct}@{domain}</strong> }}/></p>
+      <p><FormattedMessage id='onboarding.page_one.federation' defaultMessage='Mastodon is a network of independent server instances joining up to make one larger social network.' /></p>
```

Comment:
The word "instance" needs to be used since that is what is used everywhere else in the UX. And it needs to be used in a context where it is explained. Using parentheticals is typically not the best. 

How about: "Independent servers linking up to form one larger social network. We call these servers: Mastodon Instances."